.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Submitting Pull Requests
+We use [Waffle.io](https://waffle.io/ExpressGateway/express-gateway) to track work on the task board. Waffle uses keywords just like Github.
+
+Please see the checklist below and thanks again for your contribution!
+
+- [ ] in description title please use "Connect #<Github Issue>" anywhere within the description
```

Comment:
~in description title~ please use "Connect #<Github Issue>" anywhere within the description

Reply:
👍